src/main.rs

```diff
@@ -1,16 +1,7 @@
-//! Provides the main entry point to the program.
-
+mod demand;
 mod simulation;
 
-use std::env;
-use std::path::Path;
-
-/// The main entry point to the program
 fn main() {
-    let args: Vec<String> = env::args().collect();
-    if args.len() != 2 {
-        panic!("Must provide path to model configuration TOML file.");
-    }
-
-    simulation::run(Path::new(&args[1]))
+    // Initialize the simulation
+    simulation::initialize_simulation();
```

Comment:
A better separation of concerns would be for this to load the relevant data and return it, rather than loading it and the running the simulation within:

Something similar to the following, which obviously requires to update the functions and be careful about the types. 
```suggestion
    let data = simulation::initialize_simulation();
    simulation::run(data);
```

Reply:
As with PR #80, please could you put this back the way it was? We do need the user to be able to pass in the path to the `settings.toml` file.

I think @dalonsoa's structure *may* be what we go for in the longer run, but in the meantime, let's just keep the action in `simulation::run()` and let `main()` just be responsible for parsing arguments.